packages/panels/resources/lang/en/pages/auth/edit-profile.php

```diff
@@ -43,7 +43,7 @@
     'actions' => [
 
         'back' => [
```

Comment:
Not sure if we can somehow change the `back` key to `cancel` in a non-breaking way?

Reply:
We can just find and replace

Reply:
We haven't been considering that as a breaking change up until this point